File: KassenSystemAngular/ClientApp/src/app/shared/sale.model.ts
export class Sale {
  id: number;
  paymentMethod: PaymentMethod;
  datetime: Date;
  constructor() {
    this.id = 0;
    this.paymentMethod = PaymentMethod.Card;
    this.datetime = new Date();
  }
}

export enum PaymentMethod {
  Cash,
  Card
}

File: KassenSystem/Models/SaleModel.cs
using System;
using System.ComponentModel.DataAnnotations;

namespace KassenSystem.Models
{
    public enum PayWay
    {
        Card=0, Cash=1

    }
   
    public class Sale
    {
        [Key]
        public int Id { get; set; }
        [Required]
        public PayWay PayWay { get; set; }
        [Required]
        public DateTime PayTime { get; set; }
    }
}
